client/my-sites/plans/jetpack-plans/plan-upgrade/style.scss

```diff
@@ -0,0 +1,155 @@
+@import '~@wordpress/base-styles/mixins';
+@import '~@wordpress/base-styles/breakpoints';
+
+.plan-upgrade__product {
+	border-bottom: solid 2px var( --color-primary );
+
+	em {
+		font-style: normal;
+	}
+}
+
+.plan-upgrade__description {
+	max-width: 400px;
+	margin: 2rem auto 3rem;
+
+	font-size: 1rem;
+	text-align: center;
+
+	@include break-medium {
+		max-width: 650px;
+
+		font-size: 1.25rem;
+	}
+
+	@include break-wide {
+		margin-bottom: 6rem;
+	}
+
+	.with-single-reco & {
```

Comment:
When only one product is recommended, the breakpoint should be smaller.

I'm not thrilled by these style duplicates, but I haven't found a better solution yet.